### PR TITLE
Update deploy-manage-report-microsoft-defender-antivirus.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated all tools and deployment options to reflect the correct documents that were intended to be added

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9293

File: microsoft-365/security/defender-endpoint/deploy-manage-report-microsoft-defender-antivirus.md [hunk 1]
```diff
@@ -48,11 +48,11 @@ You'll also see additional links for:
...
+Group Policy and Active Directory (domain-joined)|Use a Group Policy Object to deploy configuration changes and ensure Microsoft Defender Antivirus is enabled.|Use Group Policy Objects (GPOs) to [Configure update options for Microsoft Defender Antivirus](/security/defender-endpoint/manage-protection-update-schedule-microsoft-defender-antivirus) and [Configure Windows Defender features](/security/defender-endpoint/configure-microsoft-defender-antivirus-features)|Endpoint reporting is not available with Group Policy. You can generate a list of Group Policies to determine if any settings or policies are not applied.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/deploy-manage-report-microsoft-defender-antivirus.md]:
```suggestion
Group Policy and Active Directory (domain-joined)|Use a Group Policy Object to deploy configuration changes and ensure Microsoft Defender Antivirus is enabled.|Use Group Policy Objects (GPOs) to [configure update options for Microsoft Defender Antivirus](/microsoft-365/security/defender-endpoint/manage-protection-update-schedule-microsoft-defender-antivirus) and [configure Windows Defender features](/microsoft-365/security/defender-endpoint/configure-microsoft-defender-antivirus-features).|Endpoint reporting is not available with Group Policy. You can generate a list of Group Policies to determine if any settings or policies are not applied.
```

File: microsoft-365/security/defender-endpoint/deploy-manage-report-microsoft-defender-antivirus.md [hunk 2]
```diff
@@ -48,11 +48,11 @@ You'll also see additional links for:
...
+PowerShell|Deploy with Group Policy, Microsoft Endpoint Configuration Manager, or manually on individual endpoints.|Use the [Set-MpPreference] and [Update-MpSignature] cmdlets available in the Defender module.|Use the appropriate [Get- cmdlets available in the Defender module](/powershell/module/defender)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/deploy-manage-report-microsoft-defender-antivirus.md]:
```suggestion
PowerShell|Deploy with Group Policy, Microsoft Endpoint Configuration Manager, or manually on individual endpoints.|Use the [Set-MpPreference] and [Update-MpSignature] cmdlets available in the Defender module.|Use the appropriate [Get- cmdlets available in the Defender module](/powershell/module/defender).
```

File: microsoft-365/security/defender-endpoint/deploy-manage-report-microsoft-defender-antivirus.md [hunk 3]
```diff
@@ -48,11 +48,11 @@ You'll also see additional links for:
...
+Windows Management Instrumentation|Deploy with Group Policy, Microsoft Endpoint Configuration Manager, or manually on individual endpoints.|Use the [Set method of the MSFT_MpPreference class](/previous-versions/windows/desktop/defender/set-msft-mppreference) and the [Update method of the MSFT_MpSignature class](/previous-versions/windows/desktop/defender/update-msft-mpsignature)|Use the [MSFT_MpComputerStatus](/previous-versions/windows/desktop/defender/msft-mpcomputerstatus) class and the get method of associated classes in the [Windows Defender WMIv2 Provider](/windows/win32/wmisdk/wmi-providers)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/deploy-manage-report-microsoft-defender-antivirus.md]:
```suggestion
Windows Management Instrumentation|Deploy with Group Policy, Microsoft Endpoint Configuration Manager, or manually on individual endpoints.|Use the [Set method of the MSFT_MpPreference class](/previous-versions/windows/desktop/defender/set-msft-mppreference) and the [Update method of the MSFT_MpSignature class](/previous-versions/windows/desktop/defender/update-msft-mpsignature).|Use the [MSFT_MpComputerStatus](/previous-versions/windows/desktop/defender/msft-mpcomputerstatus) class and the get method of associated classes in the [Windows Defender WMIv2 Provider](/windows/win32/wmisdk/wmi-providers).
```

File: microsoft-365/security/defender-endpoint/deploy-manage-report-microsoft-defender-antivirus.md [hunk 4]
```diff
@@ -48,11 +48,11 @@ You'll also see additional links for:
...
+Microsoft Endpoint Manager ([1](#fn1))|Use the [Endpoint Protection point site system role](/mem/configmgr/protect/deploy-use/endpoint-protection-site-role) and [enable Endpoint Protection with custom client settings](/mem/configmgr/protect/deploy-use/endpoint-protection-configure-client)|With [default and customized antimalware policies](/microsoft-365/security/office-365-security/configure-anti-malware-policies) and client management|With the default [Configuration Manager Monitoring workspace](/mem/configmgr/apps/deploy-use/monitor-applications-from-the-console) and email alerts.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/deploy-manage-report-microsoft-defender-antivirus.md]:
```suggestion
Microsoft Endpoint Manager ([1](#fn1))|Use the [Endpoint Protection point site system role](/mem/configmgr/protect/deploy-use/endpoint-protection-site-role) and [enable Endpoint Protection with custom client settings](/mem/configmgr/protect/deploy-use/endpoint-protection-configure-client).|With [default and customized antimalware policies](/microsoft-365/security/office-365-security/configure-anti-malware-policies) and client management.|With the default [Configuration Manager Monitoring workspace](/mem/configmgr/apps/deploy-use/monitor-applications-from-the-console) and email alerts.
```

File: microsoft-365/security/defender-endpoint/deploy-manage-report-microsoft-defender-antivirus.md [hunk 5]
```diff
@@ -48,11 +48,11 @@ You'll also see additional links for:
...
+Microsoft Azure|Deploy Microsoft Antimalware for Azure in the [Azure portal, by using Visual Studio virtual machine configuration, or using Azure PowerShell cmdlets](/azure/security/azure-security-antimalware#antimalware-deployment-scenarios). You can also [Install Endpoint protection in Microsoft Defender for Cloud*](/azure/defender-for-cloud/endpoint-protection-recommendations-technical)|Configure [Microsoft Antimalware for Virtual Machines and Cloud Services with Azure PowerShell cmdlets](/azure/security/azure-security-antimalware#enable-and-configure-antimalware-using-powershell-cmdlets) or [use code samples](https://gallery.technet.microsoft.com/Antimalware-For-Azure-5ce70efe)|Use [Microsoft Antimalware for Virtual Machines and Cloud Services with Azure PowerShell cmdlets](/azure/security/azure-security-antimalware#enable-and-configure-antimalware-using-powershell-cmdlets) to enable monitoring. You can also review usage reports in Azure Active Directory to determine suspicious activity, including the Possibly infected devices report and configure an SIEM tool to report on [Microsoft Defender Antivirus events][] and add that tool as an app in AAD.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/deploy-manage-report-microsoft-defender-antivirus.md]:
```suggestion
Microsoft Azure|Deploy Microsoft Antimalware for Azure in the [Azure portal, by using Visual Studio virtual machine configuration, or using Azure PowerShell cmdlets](/azure/security/azure-security-antimalware#antimalware-deployment-scenarios). You can also [Install Endpoint protection in Microsoft Defender for Cloud](/azure/defender-for-cloud/endpoint-protection-recommendations-technical).|Configure [Microsoft Antimalware for Virtual Machines and Cloud Services with Azure PowerShell cmdlets](/azure/security/azure-security-antimalware#enable-and-configure-antimalware-using-powershell-cmdlets) or [use code samples](https://gallery.technet.microsoft.com/Antimalware-For-Azure-5ce70efe).|Use [Microsoft Antimalware for Virtual Machines and Cloud Services with Azure PowerShell cmdlets](/azure/security/azure-security-antimalware#enable-and-configure-antimalware-using-powershell-cmdlets) to enable monitoring. You can also review usage reports in Azure Active Directory to determine suspicious activity, including the Possibly infected devices report, and configure an SIEM tool to report on [Microsoft Defender Antivirus events][/microsoft-365/security/defender-endpoint/troubleshoot-microsoft-defender-antivirus] and add that tool as an app in AAD.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by denisebmsft — APPROVED** [review]
Thank you so much @msbemba @JohanFreelancer9

**Pull request merged**